### PR TITLE
docs(website): Update node-troubleshooting.mdx

### LINKED ISSUE #14313: Additional Information
### Describe the bug

https://taiko.xyz/docs/reference/node-troubleshooting
When I turn a node without properly configuring the RPC, I get `error="initialize rpc clients error"`. However, this information is not written.

### Steps to reproduce

Steps to reproduce here.

### Additional context

Additional context here.

### PR DESCRIPTION
**Pull request by zakk15**
Added error and solution for RPC mistakes when running Prove/Proposer on Taiko's testnet

### CODE REVIEW

File: packages/website/pages/docs/reference/node-troubleshooting.mdx [hunk 1]
```diff
@@ -10,6 +10,12 @@ You can ignore any WARN logs.
 
 Make sure that your RPC is pointing to an Sepolia archive node, and not a full node.
 
+#### `error="initialize rpc clients error"`
+
+The RPC in your config is wrong. You have to set the ENDPOINT correctly. Please refer to the following page to set the correct RPC https://taiko.xyz/docs/guides/run-a-node/run-a-taiko-node
```
Comment (dantaik) [inline packages/website/pages/docs/reference/node-troubleshooting.mdx:15]:
```suggestion
The RPC in your config is incorrect. You have to set the ENDPOINT correctly.  Please refer to the following page to set the correct RPC https://taiko.xyz/docs/guides/run-a-node/run-a-taiko-node.
```

Comment (zakk15) [reply packages/website/pages/docs/reference/node-troubleshooting.mdx:15]: Thank you for confirmation
Wrong doesn't appear and incorrect is the correct sentence (I'm not good at English, so I made a mistake, sorry)

File: packages/website/pages/docs/reference/node-troubleshooting.mdx [hunk 2]
```diff
@@ -10,6 +10,12 @@ You can ignore any WARN logs.
 
 Make sure that your RPC is pointing to an Sepolia archive node, and not a full node.
 
+#### `error="initialize rpc clients error"`
+
+The RPC in your config is incorrect. You have to set the ENDPOINT correctly. Please refer to the following page to set the correct RPC https://taiko.xyz/docs/guides/run-a-node/run-a-taiko-node
```
Comment (dantaik) [inline packages/website/pages/docs/reference/node-troubleshooting.mdx:15]:
```suggestion
The RPC in your config is incorrect. You have to set the ENDPOINT correctly.  Please refer to the following page to set the correct RPC: https://taiko.xyz/docs/guides/run-a-node/run-a-taiko-node
```

**Review by dantaik — APPROVED** [review]

**Review by dantaik — DISMISSED** [review]

File: packages/website/pages/docs/reference/node-troubleshooting.mdx [hunk 3]
```diff
@@ -10,6 +10,12 @@ You can ignore any WARN logs.
 
 Make sure that your RPC is pointing to an Sepolia archive node, and not a full node.
 
+#### `error="initialize rpc clients error"`
```
Comment (dantaik) [inline packages/website/pages/docs/reference/node-troubleshooting.mdx:13]:
```suggestion
#### `error: "initialize rpc clients error"`
```

File: packages/website/pages/docs/reference/node-troubleshooting.mdx [hunk 4]
```diff
@@ -10,6 +10,12 @@ You can ignore any WARN logs.
...
+
+The RPC in your config is incorrect. 
You have to set the ENDPOINT correctly.
 Please refer to the following page to set the correct RPC: https://taiko.xyz/docs/guides/run-a-node/run-a-taiko-node
+
+#### `error: "failed to insert new head to L2 execution engine: missing trie node"`
+
```
Comment (d1onys1us) [inline packages/website/pages/docs/reference/node-troubleshooting.mdx:18]:
```suggestion
```

**Review by dionysuzx — APPROVED** [review]

**Comment by d1onys1us** [comment]
closes #14313

**Review by davidtaikocha — APPROVED** [review]

**Pull request merged**